### PR TITLE
add import from docano jsonl

### PR DESCRIPTION
**Pull request by DrFabach**
Add class DoccannoJsonlConverter which loads files created by doccano with entities and relationships in jsonl format.
No attribute imported from files and creation of jsonl from medkit doc need to be done

exemple  
docs = doccanno_converter.load(file_path="path/to/admin.jsonl")

### CODE REVIEW

**Comment by camila-ud** [comment]
Hello @DrFabach, thanks for this PR. We are currently working to integrate a `DoccanoInputConverter` and a `DoccanoOutputConverter` in medkit.

The internal version is almost ready, I will contact you when we have integrated it :)

File: medkit/io/doccano.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        text_anns = list()
+        
+
+        for doc_i in doccano_doc:
+            doc_ann=json.loads(doc_i)
+            anns_by_doccano_id = dict()
+            # First convert entities, then relations, finally attributes
+            # because new annotation identifier is needed
+            text = doc_ann["text"]
+            ch_sp=[m.start(0)-ind for ind, m in enumerate(re.finditer("\r", text))]
```
Comment (camila-ud) [inline medkit/io/doccano.py:157]: I don't quite understand what this exact line is doing here. As far as I've noticed, **doccano** always uses the text as reference and we can get the span of each entity using `start_offset` and `end_offset`

Do you have an example where that doesn't work?

**Comment by DrFabach** [comment]
Docanno consider "\r" token as 2 characters and medkit as one.

**Comment by camila-ud** [comment]
OK, I got it. Do you mind sending us the file where you found this problem? Thanks

**Comment by DrFabach** [comment]
Here it is !

**Comment by camila-ud** [comment]
Thank you for your time, we have found out why this can happen.
Doccano has an extra parameter: **count grapheme as one character**, in sequence labeling projects (see image). 

grapheme

This can cause **misalignment** when exporting or loading data, even within doccano. If we load an exported file from one project with the option enabled into another project without the option, the entities will not be aligned. 

For the @DrFabach  use case, we agreed to set up the project without this option. I think in medkit we should check if the character is present, and throw a warning about possible alignment problems.

**Pull request closed without merging**